src/OVAL/probes/unix/file.c

```diff
@@ -352,6 +352,8 @@ static int file_cb(const char *prefix, const char *p, const char *f, void *ptr)
 		if (se_acl == NULL) {
 			probe_item_ent_add(item, "has_extended_acl", NULL, gr_true);
 			probe_itement_setstatus(item, "has_extended_acl", 1, SYSCHAR_STATUS_DOES_NOT_EXIST);
+		} else {
+			SEXP_free(se_acl);
```

Comment:
@jan-cerny Could this be a false positive? Because `gr_true` and `gr_false` and freed during `probe_fini` on a `SEXP-vfree` call.

Reply:
@yuumasato Thanks for this great observation. This patch should be applied on master, where gr_true and gr_false are not used, but instead SEXP_number_newb is called in `has_extended_acl`. I will change the base branch.